docs/custom-transforms-and-renditions.md

```diff
@@ -100,27 +81,59 @@ performed by that particular service. It is possible to disable individual
 Local Transforms by setting the corresponding T-Engine URL property
 **localTransform.&lt;engineName>.url** value to an empty string.
 
-```properties
+```text
 localTransform.helloworld.url=
 ```
 
-### Configure a custom Local transform pipeline
+## Transformer Config
```

Comment:
Now that the json config is identical for repo and ats, I'm trying to avoid using "Local" when it applies to both.